generator/templates/Deployment/_github/workflows/deploy.demo.yml

```diff
@@ -3,14 +3,14 @@ name: Deploy [production]
 on:
   push:
     branches:
-      - 'env/production'
```

Comment:
don't change that. Production is the default, not all projects have a demo env
(and also the file name)

Reply:
But al projects have a production :D